dspace-api/src/main/java/org/dspace/xmlworkflow/state/actions/processingaction/AcceptEditRejectAction.java

```diff
@@ -34,6 +34,7 @@
 
     public static final int MAIN_PAGE = 0;
     public static final int REJECT_PAGE = 1;
+    public static final int SUBMITTER_IS_DELETED_PAGE = 2;
```

Comment:
Good catch! I changed `AcceptEditRejectAction.proccessMainPage(...)`. No it will be returned if the reject button is pressed and the submitter is set null. Will be part of our next commit.